RepoDb.Core/RepoDb/Extensions/ExpressionExtension.cs

```diff
@@ -382,7 +382,7 @@ public static object GetValue(this NewExpression expression)
             if (expression.Arguments.Count > 0)
             {
                 return Activator.CreateInstance(expression.Type,
-                    expression.Arguments.Select(arg => arg.GetValue()));
+                    expression.Arguments.Select(arg => arg.GetValue()).ToArray());
```

Comment:
Needed for supporting `== new DateOnly(2026,1,1)` in queries. Without it the constructor is not properly found.

Reply:
Interestingly that it cannot be found from the yielded result here. This is still OK for as long we have still the passing existing Integration Tests.